Microsoft.VisualStudio.Terminal/TermWindowPackage.cs

```diff
@@ -35,8 +35,7 @@ namespace Microsoft.VisualStudio.Terminal
     [Guid(TermWindowPackage.PackageGuidString)]
     [SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1650:ElementDocumentationMustBeSpelledCorrectly", Justification = "pkgdef, VS and vsixmanifest are valid VS terms")]
     [ProvideMenuResource("Menus.ctmenu", 1)]
-    [ProvideToolWindow(typeof(TermWindow), Style = VsDockStyle.Tabbed, Window = "34E76E81-EE4A-11D0-AE2E-00A0C90FFFC3")]
-    [ProvideToolWindow(typeof(ServiceToolWindow), Transient = true, MultiInstances = true, Style = VsDockStyle.Tabbed, Window = TermWindow.TermWindowGuidString)]
+    [ProvideToolWindow(typeof(TermWindow), Transient = true, MultiInstances = true, Style = VsDockStyle.Tabbed, Window = "34E76E81-EE4A-11D0-AE2E-00A0C90FFFC3")]
```

Comment:
I'm concerned about the implications of only having one tool window here. The original suggestion was just to remove the duplication of the renderer and the service window. The reason there were two windows to begin with was that a multi-instance window would not persist across sessions nor remember its last position if you closed and reopened it. The two-window solution allowed there to be a "main" terminal window that was accessible from a shortcut and menu command and that all of the temporary windows would be able to dock to its location, allowing the user to have a preference on their terminal location.

Reply:
Good point. Will fix